relbench/external/graph.py

```diff
@@ -90,6 +82,13 @@ def make_pkey_fkey_graph(
 
         col_to_stype = col_to_stype_dict[table_name]
 
+        # Remove pkey, fkey columns since they will not be used as input
+        # feature.
+        if table.pkey_col is not None:
+            col_to_stype.pop(table.pkey_col)
+        for fkey in table.fkey_col_to_pkey_table.keys():
+            col_to_stype.pop(fkey)
```

Comment:
Moved the pkey fkey removal logic in `made_pkey_fkey_graph` so that these keys get removed regardless of whether we call `get_stype_proposal()`